docs/src/pages/component-demos/grid-list/AdvancedGridList.js

```diff
@@ -0,0 +1,71 @@
+// @flow weak
+
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
```

Comment:
Just the demos (exposed to users), I haven't changed the source of the components or the regressions tests.